content/projects/gsoc/2023/project-ideas/docker-compose-build.adoc

```diff
@@ -18,27 +18,49 @@ links:
 //   gitter: "jenkinsci/plugin-installation-manager-cli-tool"
 //   draft: https://docs.google.com/document/d/1s-dLUfU1OK-88bCj-GKaNuFfJQlQNLTWtacKkVMVmHc
 ---
-// === Background
-// TBD
+=== Background
+
+We already have a link:../../../../doc/book/installing/docker/[section] dedicated to docker in the documentation, but it's too difficult to use for beginners, and even too complex to begin with. We need to provide a simple way to get started with Jenkins and Docker.
```

Comment:
`link:../../../../doc/book` can/should be `link:/doc/book` as doc is at the root.